_data-prepper/pipelines/configuration/processors/otel-metrics-raw.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: default
+title: otel metrics raw
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# otel metrics raw
+
+The `otel metrics raw` processor serializes a collection of `ExportMetricsServiceRequest` sent from the [otel-metrics-source]({{site.url}}{{site.baseurl}}//data-prepper/pipelines/configuration/sources/otel-metrics-source/) into a collection of string records.
+
+## Usage
+
+To get started, you can add the following processor to your `.yaml` configuration file:
+
+``` YAML
+processor:
+    - otel_metrics_raw_processor:
+```
+
+## Configuration
+
+The following table describes the different parameters used to configure histogram buckets and their default values. A histogram displays numerical data by grouping data into buckets. You can use histrogram buckets to view sets of events that are organized by the total event count and aggregate sum for all events. For more detailed information, see (OpenTelemetry Histograms)[https://opentelemetry.io/docs/reference/specification/metrics/data-model/#histogram].
+
+| Parameter | Default value | Description |
+| :---    | :---    | :---    |
```

Comment:
Are all of these parameters required or optional?

Reply:
@sharraj

Reply:
@Naarcha-AWS Raj has specified that these are all optional.